bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -487,11 +487,11 @@ spec:
   minKubeVersion: 1.19.0
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.56.0
+  replaces: jaeger-operator.v1.58.0
```

Comment:
Shouldn't this be `1.57.0` (the last shipped release) ?

Reply:
Yes :x

Reply:
```suggestion
  replaces: jaeger-operator.v1.57.0
```